content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -382,7 +382,7 @@ kubelet
       <td colspan="2">--feature-gates mapStringBool</td>
```

Comment:
Revert changes to this file, as it is generated. These docs will be updated as part of the 1.11 docs release process.

Reply:
@Lion-Wei Can you fix it?

Reply:
Sure,